src/pages/People.tsx

```diff
@@ -0,0 +1,75 @@
+import React, { useEffect, useState } from 'react';
+import { Loader } from '../components/Loader';
+import { getPeople } from '../api';
+import { Person } from '../types';
+import { PersonLink } from '../components/PersonLink';
+
+export const People: React.FC = () => {
+  const [people, setPeople] = useState<Person[]>();
+  const [isLoading, setIsLoading] = useState(true);
+  const [isError, setIsError] = useState(false);
+
+  useEffect(() => {
+    const gettingPeople = async () => {
+      try {
+        const peopleFromServer = await getPeople();
+
+        const peopleWithParents = peopleFromServer.map(person => ({
+          ...person,
+          father: peopleFromServer.find(p => p.name === person.fatherName),
+          mother: peopleFromServer.find(p => p.name === person.motherName),
+        }));
+
+        setPeople(peopleWithParents);
+      } catch (e) {
+        setIsError(true);
+      } finally {
+        setIsLoading(false);
+      }
+    };
+
+    gettingPeople();
+  }, []);
+
+  return (
+    <div>
+      <h1 className="title">People Page</h1>
+
+      <div className="block">
+        <div className="box table-container">
+          {isLoading ? (
+            <Loader />
+          ) : isError ? (
+            <p data-cy="peopleLoadingError" className="has-text-danger">
+              Something went wrong
+            </p>
+          ) : !people?.length ? (
+            <p data-cy="noPeopleMessage">There are no people on the server</p>
+          ) : (
+            <table
+              data-cy="peopleTable"
+              className="table is-striped is-hoverable is-narrow is-fullwidth"
+            >
+              <thead>
+                <tr>
+                  <th>Name</th>
+                  <th>Sex</th>
+                  <th>Born</th>
+                  <th>Died</th>
+                  <th>Mother</th>
+                  <th>Father</th>
+                </tr>
+              </thead>
+
+              <tbody>
+                {people.map(person => (
+                  <PersonLink key={person.name} person={person} />
```

Comment:
Using `person.name` as a key might not be unique if there are multiple people with the same name. Consider using a unique identifier, such as `person.slug` or another unique property, to ensure that each element has a unique key.